profiles/herkura.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/54028832/214070601-70731901-12c3-4223-a6ba-4851d3e4437d.png",
```

Comment:
I recommend more of an image like this. What do you think?

![image](https://user-images.githubusercontent.com/51391473/214107857-e96feb1f-eadb-49e5-8246-2f95b291293b.png)

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/214107857-e96feb1f-eadb-49e5-8246-2f95b291293b.png",
```

Reply:
Well I thought that I should cover all the aspects of my GitHub read me profile. Hence provided that image. If there is a problem then I will use the image you provided and create another PR.

Reply:
What do you say?

Reply:
The thing is that the image is way too big. We encourage having medium-sized photos. 🙂

Reply:
Sure, then I am fine with the image you have provided